util/tar.go

```diff
@@ -125,7 +125,7 @@ func sanitizedName(filename string) string {
 		filename = filename[2:]
 	}
 	filename = strings.TrimLeft(filename, "\\/.")
-	filename = strings.TrimLeft(filename, "../")
+	filename = strings.TrimLeft(filename, "./")
```

Comment:
Out of all these improvements, this one looks like it is changing functionality?
a. Old behaviour is "../my.file" -> "my.file"
b. New behaviour, it seems, would leave the string "../my.file" intact

Reply:
I believe it should not as per go doc https://golang.org/pkg/strings/#TrimLeft

> TrimLeft returns a slice of the string s with all leading Unicode code points contained in cutset removed. 

It does not matter how many characters (of the same name) are in _cutset_. It would clean all characters (unicode code points) in the cutset.

Here is the official example from godoc. https://play.golang.org/p/LBj5Z_hQpMT
`fmt.Print(strings.TrimLeft("¡¡¡Hello, Gophers!!!", "!¡"))`
gives output (despite only one "!" char listed):
`Hello, Gophers!!!`

Here are also some random example from me:
https://play.golang.org/p/6UjTznm3wnk 


I also included your example into tests and it still passes:
$ go test -v -run "TestSanitizedName" .
=== RUN   TestSanitizedName
--- PASS: TestSanitizedName (0.00s)
PASS
ok      github.com/vmware/go-vcloud-director/v2/util    0.015s

Reply:
Ergh, that's one trim function definition to trip on (I mean the way it uses cutsets)! Thanks for clarifying.